src/DeviceInput/InputDevices/deviceSourceManager.ts

```diff
@@ -43,6 +43,23 @@ export class DeviceSourceManager {
      */
     public getDevices: () => ReadonlyArray<DeviceSource<DeviceType>>;
 
+    /**
+     * Enable handling of input events
+     * @hidden
+     */
+    public enableEvents: () => void;
+
+    /**
+     * Disable handling of input events
+     * @hidden
+     */
+    public disableEvents: () => void;
+
+    /**
+     * Dispose of DeviceSourceManager
+     */
+    public dispose: () => void;
```

Comment:
Probably makes more sense to extend `IDisposable`

Reply:
I can implement IDisposable but that'd also require having this method because it's an interface (unless I'm misunderstanding this comment)

Reply:
Yes, you are right. Extending from IDisposable will allow people to see quickly that you can dispose without searching. It's not a big deal.